src/components/ProteinViewer/grid.css

```diff
@@ -9,10 +9,12 @@
 }
 
 .protvista-grid .view-options-wrap {
+  margin-top: 30px;
   display: flex;
-  align-items: baseline;
+  align-items: center;
   justify-content: space-between;
```

Comment:
Remove this rule so we can push the `Options` component to the right.

```suggestion
```

Then add the following rule under `.view-options { ... }` in `src/components/ProteinViewer/Options/styles.css`:

```css
  margin-left: 0 0.5rem 0 auto;
```